neuralmonkey/encoders/transformer.py

```diff
@@ -128,9 +139,22 @@ def __init__(self,
                 "If provided, the target space ID should be between 0 and 31. "
                 "Was: {}".format(self.target_space_id))
 
+        if (input_for_cross_attention is None) ^ (n_cross_att_heads is None):
+            raise ValueError(
+                "Either both input_for_cross_attention and n_cross_att_heads "
+                "must be provided or none of them.")
+
+        if input_for_cross_attention is not None:
```

Comment:
vzít se selfu

Reply:
proč?

Reply:
Děláme to tak i jinde. Nevim, jestli je to pythonic nebo ne. Může to mít dopady na věci kolem dědičnosti.

Reply:
Nemusíš to dělat, je to jedno.